snuba/consumers/consumer.py

```diff
@@ -402,6 +492,9 @@ def close(self) -> None:
                 try:
                     step.close()
```

Comment:
You do not clean up `self.__messages` because this colelctor is created and destroyed at every batch, is that correct ?

Reply:
I _think_ so... but I could also reset `self.__messages` in the join method